packages/name/src/formatName.ts

```diff
@@ -0,0 +1,37 @@
+import {EASTERN_NAME_ORDER_FORMATTERS} from './constants';
+import {hasEasternNameOrderFormatter} from './utilities';
+
+export const formatName = Object.assign(
```

Comment:
I think it would be better to have `formatName` as a method only and expose `hasEasternNameOrderFormatter` as another top-level method. Then you could move that method into its own file and maybe not have a separate file in `utilities`

Reply:
I'm basing my thinking on the possibility that other repos might call this method independently outside of `formatName`. In our repos, there's only one reference I could find [here](https://github.com/Shopify/pos-channel/blob/5e04d06f5c3bd2fec254961873f7c38524217d4d/app/ui/sections/Staff/components/AccountInformation/AccountInformation.tsx#L118), where it's being used to decide what order the personal name fields should be displayed in.

I otherwise don't think it's necessary to strongly associate it with `formatName`; you can tell from context (and the README) that it pertains to personal name, not business name, for example.

Reply:
oh I stand corrected! I see now that this is being used to render names in the correct order in cases where we don't use format name. thanks for clarifying! 

Do only "eastern" dialects have last name then first name ordering? do any not have this ordering? do any non-eastern dialects have this ordering? Given these questions... do you think `hasFamilyNameFirstPersonalNameOrdering` or something more along those lines would be better?